src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+
+  .body {
+    font-family: "Roboto", sans-serif;
+  }
+}
```

Comment:
these styles will be better to move to normalize.scss file
and move styles for body outside of * selector